### PR TITLE
seltest screenshots

### PR DESCRIPTION
**Pull request by danvk**
For such a visual product, it's about time we did this.

[Review on Reviewable](https://reviewable.io/reviews/hammerlab/pileup.js/342)

### CODE REVIEW

File: screenshots/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+The screenshots in this directory are updated manually on a best-effort basis.
+
+To generate them, run:
+
+   pip install seltest
+
+   sel -b chrome update screenshots
+
+You'll need to install chrome webdriver for this to work.
```
Comment (ihodes) [inline screenshots/README.md]: could link to https://sites.google.com/a/chromium.org/chromedriver/

File: screenshots/test.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+PILEUP_RENDERED_JS = '''
+return document.querySelector('.track.pileup canvas') && document.querySelector('.track.pileup canvas').height > 100
+'''
+
+def _pileup_is_rendered(driver):
+    return driver.execute_script(PILEUP_RENDERED_JS)
+
+
+def wait_for_pileup(driver):
+    driver.implicitly_wait(0)
```
Comment (ihodes) [inline screenshots/test.py]: Hmm, for my own edification, why do you need to wait for 0 seconds? 

Definitely need to add this to seltest as a decorator.

File: screenshots/test.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    WebDriverWait(driver, 120).until(_pileup_is_rendered, 'pileup timed out')
+    driver.implicitly_wait(120)
+
+
+class Pileup(Base):
```
Comment (ihodes) [inline screenshots/test.py]: filed https://github.com/hammerlab/seltest/issues/62 as well to make this easier in the future

**Comment by ihodes** [comment]
LGTM, just one suggestion re: adding a link to the Chrome driver. Sorry getting seltest running was so painful!

File: screenshots/test.py (hunk 2 above)
Comment (danvk) [inline screenshots/test.py]: I cribbed that from [here][1]. I'm not sure why, but it was necessary—the test didn't wait properly if I didn't include it.

[1]: https://github.com/hammerlab/seltest/blob/c73caeee879a790a6a21396e502710802f0430f3/seltest/seltest.py#L188-L192

File: screenshots/README.md (hunk 1 above)
Comment (danvk) [inline screenshots/README.md]: Done.

File: screenshots/test.py (hunk 3 above)
Comment (danvk) [inline screenshots/test.py]: 👍

**Comment by danvk** [comment]
There's still one issue I haven't tracked down... when I try to take screenshots in Firefox, the page gets stuck loading the pileup track. I'm not sure if it's a pileup issue or a seltest issue. We can try to debug tomorrow.

**Comment by danvk** [comment]
FWIW, there's no reason this change should affect code coverage: there must be some flakiness in which code paths are executed.

File: screenshots/test.py (hunk 2 above)
Comment (ihodes) [inline screenshots/test.py]: Hah, what a clever person to have debugged that in the past…! Weird.

**Pull request merged**